index.html

```diff
@@ -672,10 +862,10 @@ <h2 class="feedback-card__username">Adam</h2>
         </div>
       </section>
 
-      <!-- Fifth-section -->
-      <section class="fifth-section">
-        <div class="container fifth-section__container">
-          <div class="fifth-section__subscribe-block subscribe-block">
+      <!-- subscribe -->
+      <section class="subscribe">
+        <div class="container subscribe__container">
```

Comment:
```suggestion
        <div class="container">
```
Think one more time about the structure of your sections. This may be redone.